src/js/containers/covid19/helpers/scrollHelper.jsx

```diff
@@ -0,0 +1,23 @@
+export const isElementVisible = (ref) => {
+    const rect = ref.current.getBoundingClientRect();
+    const verticallyVisible = (rect.top <= window.innerHeight) && ((rect.top + rect.height) >= 0);
+    const horizontallyVisible = (rect.left <= window.innerWidth) && ((rect.left + rect.width) >= 0);
+    return (verticallyVisible && horizontallyVisible);
+};
+
+
+// this function does not work in IE11
+export const scrollIntoView = (loading, error, wrapperRef, wrapperReadyRef, margin, scrollIntoViewOptions, moreOptionsTabsRef) => {
+    const isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
+
+    if ((loading || error) && !isIE11 && wrapperRef.current && moreOptionsTabsRef.current && isElementVisible(wrapperRef) && window.pageYOffset > 0) {
```

Comment:
To me, it simplifies the conditional logic if we can add a nested if here so

```javascript
else if (loading || error) {
// handle loading or error
return
}
``` 

and 

```javascript
else if (conditionals) {
// safely assume loading and error are both false here to simplify logic
}
``` 

One more thing I'd say is can we store some of these ands as variables just for the sake of readability?

For example


```javascript
const isElementVisibleAndRoomExists = (
  isElementVisible(wrapperReadyRef) &&
  window.pageYOffset > 0
)
```

or something like that.